### PR TITLE
New feature project popup

### PR DESCRIPTION
**Pull request by shuhad786**
- Updated the Project-popup.js to enable the popup window through JS objects.
- Partner assisted with closing popup window.
- Updated the HTML classes buttons and created a new section to store popup functions.
- Updated the CSS to style the popup window according to the Figma template.
- Updated the README.md to include the new framework and technologies.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -350,7 +350,7 @@ <h2 class="project-title">Professional Art <br> Printing Data</h2>
           Ruby
         </li>
       </ul>
-      <button type="button" onclick="" class="project-button">See Project</button>
+      <button type="button" onclick="" class="Mproject-button-4">See Project</button>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly populate the list of projects dynamically using javascript

Screen Shot 2022-09-07 at 15 54 13

File: Project-popup.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  text2: 'Lorem Ipsum is simply dummy text of the printing and <br> typesetting industry. Lorem Ipsum has been the <br> industrys standard dummy text ever since the 1500s, <br> when an unknown printer took a galley of type and <br> scrambled it 1960s with the releaLorem Ipsum is simply <br> dummy text of the printing and typesetting industry. <br> Lorem Ipsum has been the industrys standard dummy <br> text ever since the 1500s, when an unknown printer <br> took a galley of type and scrambled it 1960s with the <br> releax map lapora verita.',
+  techstack1: 'html',
+  techstack2: 'Bootstrap',
+  techstack3: 'Ruby on rails',
+  button1: 'See Live',
+  button2: 'See Source',
+  button1Img: './images/popup-icon-see project.png',
+  button2Img: './images/popup-icon-github.png',
+}];
+
```
Comment (aimemalaika) [inline Project-popup.js:18]: - [ ] When you click the button on see project kindly populate the popup with the content of the selected project
Screen Shot 2022-09-07 at 15 55 50

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @shuhad786 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @shuhad786, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**